BatchApex/BatchApex.cls

```diff
@@ -0,0 +1,119 @@
+/**
+ * Collection of Closed Won Opportunities 
+ * The Primary contact associated with an account needs to know which all are the 
+ * closed won opportunities for his account per day. 
+ * Create a process which will run everyday in the midnight & collect all the 
+ * Closed won opportunities of that account in a day & send an email to Primary Contact. 
+ * The email body should contain the Opportunity Name & it's respective amount 
+ * with the aggregated amount in the end. 
+ */
+global with sharing class BatchApex implements Database.Batchable<Sobject>, Database.Stateful, 
+    Schedulable {
+    map<String,WrapperClass> accountOpportunityMap;
+    public BatchApex() {
+        accountOpportunityMap = new Map<String,WrapperClass>();
```

Comment:
In this map we store the account and its related opportunity list  and the sum of their amounts